### ISSUE TITLE
GDrive reports that the poster does not exist

### ISSUE THREAD

**Issue by kgonia** [nerfstudio-project/nerfstudio#3052]
When attempting to use the ns-download-data nerfstudio --capture-name=poster command to download data, I encounter an error indicating that the file's public link cannot be retrieved. The specific error message suggests issues with the file's permission settings or an excessive number of accesses. However, upon directly accessing the provided Google Drive link, the platform reports that the file does not exist.

- Execute the command ns-download-data nerfstudio --capture-name=poster in the terminal.
- Encounter the error message regarding the inability to retrieve the public link of the file.
- Directly visit the Google Drive link: https://drive.google.com/file/d/1dmjWGXlJnUxwosN6MVooCDQe970PkD-1/view?usp=sharing
- Observe the message indicating the file does not exist.

**Comment by Zzz212zzZ** [comment]
Me too, I got into the same issue. To make a supplement, below is the error message:

```
Traceback (most recent call last):
.................
  File "/home/user/.local/lib/python3.10/site-packages/gdown/download.py", line 264, in download
    raise FileURLRetrievalError(message)
gdown.exceptions.FileURLRetrievalError: Failed to retrieve file url:

        Cannot retrieve the public link of the file. You may need to change
        the permission to 'Anyone with the link', or have had many accesses.
        Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq.

You may still be able to access the file from the browser:

        https://drive.google.com/uc?id=1dmjWGXlJnUxwosN6MVooCDQe970PkD-1
```

**Comment by Zzz212zzZ** [comment]
Seems like the poster URL link is down. One feasible solution is replacing it with another available dataset. You can all see the available options [here](https://docs.nerf.studio/reference/cli/ns_download_data.html#options_repeat3). I tried the `s-download-data nerfstudio --capture-name=sculpture` and it worked fine.

**Comment by ginazhouhuiwu** [comment]
Hi! This should have been resolved #3045. Try pulling from main again perhaps?
